smart_home/src/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(smart_home main.cc services/temperature_service.cc devices/device.cc sensors/sensor.cc)
+
+target_include_directories(smart_home PUBLIC
+${CMAKE_CURRENT_SOURCE_DIR}/../include)
```

Comment:
Unless there's a very specific use case that requires a separate CMakeLists.txt file, there should be only one CMakeLists.txt file in the project root